lib/tables.c

```diff
@@ -117,6 +117,7 @@ typedef struct {
     void **array_dest;
     table_size_t *len_dest;
     table_size_t len_offset;
+    table_size_t len_multiplier;
```

Comment:
added this multiplier for the `spatial_dimension` of `individual.location`

Reply:
Ditto redundant here.